### ISSUE TITLE
Is it normal to encounter errors when using the low-level planner in the environment?

### ISSUE THREAD

**Issue by shengyin1224** [lbaa2022/LLMTaskPlanning#5]
Thanks for your wonderful work! I am using your codes in thor_connector.py which serve as low level planner for actions like 'navigate to ', 'put'. Now when I find sth, I always encounter the warning like below:

![截屏2024-08-31 下午9 45 30](https://github.com/user-attachments/assets/6dfc3dcd-79e4-44fa-8e3e-b8de8e263301)

Is it normal? If the low level planner cannot complete the correct steps, the result of benchmark maybe not accurate?

**Comment by shengyin1224** [comment]
Thanks for your reply! I think maybe the use of ai2thor-5.0 causes the error. Could you please tell me which parts needs to be modified if I use different version?

**Comment by youngwoo-yoon** [comment]
ALFRED dataset is only compatible with ai2thor 2.1, so you cannot use other versions.

**Comment by shengyin1224** [comment]
Thank you for the reply! Actually, I want to use Lota-Bench in other dataset which is created in Ai2thor-5.0, so I encounter the problem in low-level planner.

**Comment by youngwoo-yoon** [comment]
Then I think you should implement a new low-level planner (or update the current alfred low-level planner). We would welcome your contributions if you want to merge your implementation to this repository.

**Comment by shengyin1224** [comment]
Yeah, I am now modifying the low-level planner based on your implementation and it seems that 'find' or 'put' has some problems and the agent always collide with objects in the 5.0 version. I wonder the implementation of your 'find' action was through a lot of trials or you created it based on other work like ALFRED? 
![截屏2024-09-02 下午10 58 17](https://github.com/user-attachments/assets/43d8bd7a-f7cf-4a83-b6db-70dc634b2163)

**Comment by youngwoo-yoon** [comment]
We implemented this through many trial-and-error. The primary goal was to evaluate the high-level planner, so the low-level planner has some unrealistic logic, for example, the agent attempts several times for a successful pickup.

**Comment by shengyin1224** [comment]
I see, thank you for your response. I'll adjust the test according to the new version, but I feel that implementing a low-level planner is still quite challenging

**Comment by shengyin1224** [comment]
hello! I want to ask if 'find' action in your implementation makes sure that the agent will be finally close to the target object and the agent is standing facing the target object？

**Comment by youngwoo-yoon** [comment]
Yes, but it may not be facing the right direction because there is no information about the front of the object. For example, we tried to make the agent face the front of a fridge by guessing, but sometimes it ends up facing the side panel instead, causing the attempt to open the fridge to fail.
